README.md

```diff
@@ -1,6 +1,6 @@
 <h1 align="center">Hacktoberfest</h1>
 <p align="center">
-    <img src="./assets/logo.png"><br>
+    <img src="./assets/hack_fest.gif"><br>
```

Comment:
The GIF file contains some of the older logos of Hacktoberfest. I would personally not recommend this

Reply:
Okay. Will add for 2019. Thanks.